app/javascript/utils/form/final_form/Field.js

```diff
@@ -194,6 +194,39 @@ export const CheckboxField = ({ fieldName, label, blockNote, validation, id, ari
   </Field>
 )
 
+export const TextCheckboxField = ({ fieldName, label, blockNote, validation, id, ariaLabelledby, form, trueValue = 'Yes', falseValue = 'No', initialValue = null }) => {
```

Comment:
Can you add a comment to this component that explains how this is different from the `CheckboxField` component?
something like
```
Checkbox that is backed by a string value (ex: has_developental_disability: "Yes"/"No")
```